File: Source/UnitTests/TestCode/AccessorRefactoringTest/NotDeclaredAccessor.cs
namespace Test
{
	public abstract class Person
	{
		public abstract String Name
		{
			get;
			set;
		}
	}

	public class Staff : Person
	{
		private String name;
		public String Name
		{
			get{return name;}
			set{ throw new System.NotSupportedException();}
		}
	}

	public class Actor : Person
	{
		private String name;
		public String Name
		{
			get{return name;}
			set{ this.name = value;}
		}
	}
}

File: Source/UnitTests/TestCode/AccessorRefactoringTest/NotDeclaredAccessor.java
package Test;
public abstract class Person
{
	public abstract String getName();
}

public class Staff extends Person
{
	private String name;
	public String getName()
	{
		return name;
	}
}

public class Actor extends Person
{
	private String name;
	public String getName()
	{
		return name;
	}

	public void setName(String newName)
	{
		this.name = newName;
	}
}